core/modules/node/node.types.inc

```diff
@@ -312,18 +312,34 @@ function node_type_form($form, &$form_state, $type = NULL) {
     '#default_value' => $type->settings['node_submitted'],
     '#description' => t('Author username and publish date will be displayed.'),
   );
+  $node_submitted_state = array(
+    'visible' => array(
+      'input[name="node_submitted"]' => array('checked' => TRUE),
+    ),
+  );
+  $form['display']['node_submitted_format'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Author/date format'),
+    '#default_value' => $type->settings['node_submitted_format'],
```

Comment:
```suggestion
    '#default_value' => filter_xss_admin($type->settings['node_submitted_format']),
```

Reply:
Turns out this is not needed because `token_replace()` defaults to using `$options['sanitize'] = TRUE`.